homeassistant/components/qbittorrent/strings.json

```diff
@@ -48,5 +48,42 @@
         "name": "All torrents"
       }
     }
+  },
+  "services": {
+    "get_torrents": {
+      "name": "Get torrents",
+      "description": "Gets a list of current torrents",
+      "fields": {
+        "device_id": {
+          "name": "[%key:common::config_flow::data::device%]",
+          "description": "Which service to grab the list from"
```

Comment:
"grab the list" is too colloquial English and not correctly explained what it's referring to.